doc/source/contribution/report.rst

```diff
@@ -26,3 +26,52 @@ Report a bug to the mailing list
 
 Mroonga project has :doc:`/community` for discussing about Mroonga.
 Please send an Email that describes a bug.
+
+How to collect data for reporting a reproducible bug
+====================================================
+
+There may be a case that Mroonga doesn't work as expected. It may be a crash bug or
+SQL returns unexpected search results. To solve such a issues with you, We need a
+reproducible bug report.
+
+For a reproducible bug report, the following information is required at least.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Description
+   * - Environment information
+     - Installed packages information to detect what version is used.
+       rpm::
+
+           rpm -qa > INSTALLED.txt
+
+       deb::
+
+           dpkg -l > INSTALLED.txt
+
+       On Windows, show us what zip package is used.
+   * - Executing query which triggers the issue
+     - It shows what you did.
+   * - MySQL error log
+     - It may include backtrace information why crashed unexpectedly.
+   * - groonga.log
+     - It may include low level error information.
+       It may exists on ``/var/lib/mysql/groonga.log`` for example.
+
+
+There is a case that it is not enough to reproduce a bug with above information. For such a case, additional information is required.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Description
+   * - MySQL dump
+     - It may be required to construct same database from scratch.
```

Comment:
We should show generally output place of this dump like an explanation of groonga.log.

Reply:
It depends user operation.